### PR TITLE
Tidy up code

### PR DESCRIPTION
**Pull request by vananrajaindern**
Tidy up code to remove repetition of the term as suggested by class in presentation.

" let(:user) { create(:user) }"

### CODE REVIEW

File: spec/controllers/tweets_controller_spec.rb [hunk 1]
```diff
@@ -30,12 +30,11 @@
 
   describe 'POST #create' do
 
-    let(:user) { create(:user) }
-
     before do
       sign_in user
 
       post :create, params: { tweet: params }
+      git
```
Comment (gohdaniel15) [inline spec/controllers/tweets_controller_spec.rb:37]: Guess this is where it shouldn't be 😅 Hence the tests are failing.

**Review by gohdaniel15 — APPROVED** [review]
Fix the tests and 🎈

**Comment by vananrajaindern** [comment]
Will review this code again

**Comment by vananrajaindern** [comment]
Closing this as the required change is made in separate thread by @ssoonsl

**Pull request closed without merging**